### ISSUE TITLE
`KeyError` in ActorAnalysis

### ISSUE THREAD

**Issue by f-hafner** [navigating-stories/orange-story-navigator#39]
##### Story Navigator version
0.0.18 (commit https://github.com/navigating-stories/orange-story-navigator/commit/8732988e67920716c2eef3eef0af7d6e936da74f)

##### Orange version
3.36.2

##### Expected behavior
Can display the actor analysis for story 28 (contact me for title) without error

##### Actual behavior
When clicking on the story, I get the following error:
```bash

Exception:
KeyError: '27'
Module:
storynavigation.widgets.OWSNActorAnalysis:803
Widget Name:
Actors
Widget Module:
storynavigation.widgets.OWSNActorAnalysis:763
Widget Scheme:
<?xml version='1.0' encoding='utf-8'?> <scheme version="2.0" title="" description=""> <nodes> <node id="0" name="Import Documents" qualified_name="orangecontrib.text.widgets.owimportdocuments.OWImportDocuments" project_name="Orange3-Text" version="" title="Import Documents" position="(137.0, 591.0)" /> <node id="1" name="Elements" qualified_name="storynavigation.widgets.OWSNTagger.OWSNTagger" project_name="storynavigator" version="" title="Elements" position="(319.0, 504.0)" /> <node id="2" name="File" qualified_name="Orange.widgets.data.owfile.OWFile" project_name="Orange3" version="" title="File" position="(155.0, 431.0)" /> <node id="3" name="Data Table" qualified_name="Orange.widgets.data.owtable.OWTable" project_name="Orange3" version="" title="Data Table" position="(434.0, 505.0)" /> <node id="4" name="Actors" qualified_name="storynavigation.widgets.OWSNActorAnalysis.OWSNActorAnalysis" project_name="storynavigator" version="" title="Actors" position="(625.0, 629.0)" /> </nodes> <links> <link id="0" source_node_id="0" sink_node_id="1" source_channel="Corpus" sink_channel="Stories" enabled="true" source_channel_id="data" sink_channel_id="stories" /> <link id="1" source_node_id="2" sink_node_id="1" source_channel="Data" sink_channel="Custom tags" enabled="true" source_channel_id="data" sink_channel_id="custom_tag_dict" /> <link id="2" source_node_id="1" sink_node_id="3" source_channel="Story elements" sink_channel="Data" enabled="true" source_channel_id="dataset_level_data" sink_channel_id="data" /> <link id="3" source_node_id="3" sink_node_id="4" source_channel="Selected Data" sink_channel="Story elements" enabled="true" source_channel_id="selected_data" sink_channel_id="story_elements" /> <link id="4" source_node_id="0" sink_node_id="4" source_channel="Corpus" sink_channel="Stories" enabled="true" source_channel_id="data" sink_channel_id="stories" /> </links> <annotations /> <thumbnail /> <node_properties> <properties node_id="0" format="pickle">gASVCQQAAAAAAAB9lCiMEmNvbnRyb2xBcmVhVmlzaWJsZZSIjAtjdXJyZW50UGF0aJSMUC9ob21l L2ZsYXZpby9yZXBvc2l0b3JpZXMvcHJvamVjdHMvbmF2aWdhdGluZy1zdG9yaWVzL2RhdGEvc3Vi c2V0LWNvcm9uYS1kYXRhc2V0lIwIbGVtbWFfY2KUiIwGbmVyX2NilImMBnBvc19jYpSJjAxyZWNl bnRfcGF0aHOUXZQojB5vcmFuZ2V3aWRnZXQudXRpbHMuZmlsZWRpYWxvZ3OUjApSZWNlbnRQYXRo lJOUKYGUfZQojAdhYnNwYXRolGgDjAZwcmVmaXiUTowHcmVscGF0aJROjAV0aXRsZZSMAJSMBXNo ZWV0lGgSjAtmaWxlX2Zvcm1hdJROdWJoCymBlH2UKGgOjE4vaG9tZS9mbGF2aW8vcmVwb3NpdG9y aWVzL3Byb2plY3RzL25hdmlnYXRpbmctc3Rvcmllcy9kYXRhL2Z1bGwtY29yb25hLWRhdGFzZXSU aA9OaBBOaBFoEmgTaBJoFE51YmgLKYGUfZQoaA6MFS9ob21lL2ZsYXZpby9kYXRhc2V0c5RoD05o EE5oEWgSaBNoEmgUTnViZYwLcmVjZW50X3VybHOUXZSME3NhdmVkV2lkZ2V0R2VvbWV0cnmUQ0IB 2dDLAAMAAAAABawAAAHSAAAHfAAAA0MAAAWsAAAB9wAAB3wAAANDAAAAAAAAAAANcAAABawAAAH3 AAAHfAAAA0OUjAZzb3VyY2WUSwCMC19fdmVyc2lvbl9flEsBjBBjb250ZXh0X3NldHRpbmdzlF2U KIwVb3Jhbmdld2lkZ2V0LnNldHRpbmdzlIwHQ29udGV4dJSTlCmBlH2UKIwGdmFsdWVzlH2UKIwI bGFuZ3VhZ2WUjAVEdXRjaJRoIEsBdYwOZG9jdW1lbnRzX2hhc2iUighKfnv+F9QXu3ViaCUpgZR9 lChoKH2UKIwIbGFuZ3VhZ2WUjAVEdXRjaJRoIEsBdWgsigiFz8tXD5aRyHViaCUpgZR9lChoKH2U KIwIbGFuZ3VhZ2WUjAVEdXRjaJRoIEsBdWgsigiMLcTu9dFhCHViaCUpgZR9lChoKH2UKIwIbGFu Z3VhZ2WUjAVEdXRjaJRoIEsBdWgsiggZUz4oOYaP7HViaCUpgZR9lChoKH2UKIwIbGFuZ3VhZ2WU jAVEdXRjaJRoIEsBdWgsiggnLrJGlKb8V3ViaCUpgZR9lChoKH2UKIwIbGFuZ3VhZ2WUjAVEdXRj aJRoIEsBdWgsigiM/VjvihANR3ViaCUpgZR9lChoKH2UKIwIbGFuZ3VhZ2WUjAVEdXRjaJRoIEsB dWgsighmDSvGFi+XOXViZXUu </properties> <properties node_id="1" format="literal">{'autocommit': True, 'controlAreaVisible': True, 'savedWidgetGeometry': b'\x01\xd9\xd0\xcb\x00\x03\x00\x00\x00\x00\x03\xaa\x00\x00\x00~\x00\x00\x06)\x00\x00\x02\x82\x00\x00\x03\xaa\x00\x00\x00\xa3\x00\x00\x06)\x00\x00\x02\x82\x00\x00\x00\x00\x00\x00\x00\x00\rp\x00\x00\x03\xaa\x00\x00\x00\xa3\x00\x00\x06)\x00\x00\x02\x82', '__version__': 2, 'context_settings': []}</properties> <properties node_id="2" format="pickle">gASVeAsAAAAAAAB9lCiMEmNvbnRyb2xBcmVhVmlzaWJsZZSIjAxyZWNlbnRfcGF0aHOUXZQojB5v cmFuZ2V3aWRnZXQudXRpbHMuZmlsZWRpYWxvZ3OUjApSZWNlbnRQYXRolJOUKYGUfZQojAdhYnNw YXRolIyTL2hvbWUvZmxhdmlvL3JlcG9zaXRvcmllcy9wcm9qZWN0cy9uYXZpZ2F0aW5nLXN0b3Jp ZXMvb3JhbmdlLXN0b3J5LW5hdmlnYXRvci9vcmFuZ2Vjb250cmliL3N0b3J5bmF2aWdhdGlvbi9y ZXNvdXJjZXMvZHV0Y2hfaGFsbGlkYXlfYWN0aW9uX2xpc3QuY3N2lIwGcHJlZml4lE6MB3JlbHBh dGiUTowFdGl0bGWUjACUjAVzaGVldJRoDowLZmlsZV9mb3JtYXSUTnViaAYpgZR9lChoCYxJL2hv bWUvZmxhdmlvL3JlcG9zaXRvcmllcy9wcm9qZWN0cy9uYXZpZ2F0aW5nLXN0b3JpZXMvZGF0YS9v dGhlci9kaWN0LmNzdpRoC05oDE5oDWgOaA9oDmgQTnViaAYpgZR9lChoCYyHL2hvbWUvZmxhdmlv L3JlcG9zaXRvcmllcy9wcm9qZWN0cy9uYXZpZ2F0aW5nLXN0b3JpZXMvb3JhbmdlLXN0b3J5LW5h dmlnYXRvci8udmVudi9saWIvcHl0aG9uMy45L3NpdGUtcGFja2FnZXMvT3JhbmdlL2RhdGFzZXRz L2lyaXMudGFilGgLjA9zYW1wbGUtZGF0YXNldHOUaAyMCGlyaXMudGFilGgNaA5oD2gOaBBOdWJo BimBlH2UKGgJjIovaG9tZS9mbGF2aW8vcmVwb3NpdG9yaWVzL3Byb2plY3RzL25hdmlnYXRpbmct c3Rvcmllcy9vcmFuZ2Utc3RvcnktbmF2aWdhdG9yLy52ZW52L2xpYi9weXRob24zLjkvc2l0ZS1w YWNrYWdlcy9PcmFuZ2UvZGF0YXNldHMvdGl0YW5pYy50YWKUaAtoF2gMjAt0aXRhbmljLnRhYpRo DWgOaA9oDmgQTnViaAYpgZR9lChoCYyKL2hvbWUvZmxhdmlvL3JlcG9zaXRvcmllcy9wcm9qZWN0 cy9uYXZpZ2F0aW5nLXN0b3JpZXMvb3JhbmdlLXN0b3J5LW5hdmlnYXRvci8udmVudi9saWIvcHl0 aG9uMy45L3NpdGUtcGFja2FnZXMvT3JhbmdlL2RhdGFzZXRzL2hvdXNpbmcudGFilGgLaBdoDIwL aG91c2luZy50YWKUaA1oDmgPaA5oEE51YmgGKYGUfZQoaAmMkC9ob21lL2ZsYXZpby9yZXBvc2l0 b3JpZXMvcHJvamVjdHMvbmF2aWdhdGluZy1zdG9yaWVzL29yYW5nZS1zdG9yeS1uYXZpZ2F0b3Iv LnZlbnYvbGliL3B5dGhvbjMuOS9zaXRlLXBhY2thZ2VzL09yYW5nZS9kYXRhc2V0cy9oZWFydF9k aXNlYXNlLnRhYpRoC2gXaAyMEWhlYXJ0X2Rpc2Vhc2UudGFilGgNaA5oD2gOaBBOdWJoBimBlH2U KGgJjJEvaG9tZS9mbGF2aW8vcmVwb3NpdG9yaWVzL3Byb2plY3RzL25hdmlnYXRpbmctc3Rvcmll cy9vcmFuZ2Utc3RvcnktbmF2aWdhdG9yLy52ZW52L2xpYi9weXRob24zLjkvc2l0ZS1wYWNrYWdl cy9PcmFuZ2UvZGF0YXNldHMvYnJvd24tc2VsZWN0ZWQudGFilGgLaBdoDIwSYnJvd24tc2VsZWN0 ZWQudGFilGgNaA5oD2gOaBBOdWJoBimBlH2UKGgJjIYvaG9tZS9mbGF2aW8vcmVwb3NpdG9yaWVz L3Byb2plY3RzL25hdmlnYXRpbmctc3Rvcmllcy9vcmFuZ2Utc3RvcnktbmF2aWdhdG9yLy52ZW52 L2xpYi9weXRob24zLjkvc2l0ZS1wYWNrYWdlcy9PcmFuZ2UvZGF0YXNldHMvem9vLnRhYpRoC2gX aAyMB3pvby50YWKUaA1oDmgPaA5oEE51YmWMC3JlY2VudF91cmxzlF2UjBNzYXZlZFdpZGdldEdl b21ldHJ5lENCAdnQywADAAAAAAONAAAB5gAABeQAAASHAAADjQAAAgsAAAXkAAAEhwAAAAAAAAAA DXAAAAONAAACCwAABeQAAASHlIwLc2hlZXRfbmFtZXOUfZSMBnNvdXJjZZRLAIwDdXJslGgOjA1k b21haW5fZWRpdG9ylH2UjAtfX3ZlcnNpb25fX5RLAYwQY29udGV4dF9zZXR0aW5nc5RdlIwVb3Jh bmdld2lkZ2V0LnNldHRpbmdzlIwHQ29udGV4dJSTlCmBlH2UKIwGdmFsdWVzlH2UKIwJdmFyaWFi bGVzlF2UaDV9lGhBXZQoXZQojAVyZWFsbZSMFE9yYW5nZS5kYXRhLnZhcmlhYmxllIwOU3RyaW5n VmFyaWFibGWUk5RLAowVQmVpbmcsIERvaW5nLCBTZW5zaW5nlIllXZQojAdwcm9jZXNzlGhJSwKM PUJlaGF2aW9yYWwsIEV4aXN0ZW50aWFsLCBNYXRlcmlhbCwgTWVudGFsLCBSZWxhdGlvbmFsLCBW ZXJiYWyUiWVdlCiMBnByb3N1YpRoSUsCjIRCZWhhdmlvcmFsLCBFeGlzdGVudGlhbCwgTWF0ZXJp YWwsIE1lbnRhbDogQ29nbml0aXZlLCBNZW50YWw6IERlc2lkZXJhdGl2ZSwgTWVudGFsOiBFbW90 aXZlLCBNZW50YWw6IFBlcnNwZWN0aXZlLCBSZWxhdGlvbmFsLCBWZXJiYWyUiWVdlCiMA3N1YpRo SUsCjLNBY3RzIHZlcmJhbGx5LCBCb2RpbHkgcG9zdHVyZXMsIENpcmN1bXN0YW50aWFsLCBDb2du aXRpdmUsIENyZWF0aXZlLCBEZXNpZGVyYXRpdmUsIERpc3Bvc2l0aXZlLCBFbW90aXZlLCBFeGlz dGVudGlhbCwgSW50ZW5zaXZlLCBQZXJzcGVjdGl2ZSwgUGh5c2lvbG9naWNhbCwgUG9zc2Vzc2l2 ZSwgVmVyYmFsIGFjdJSJZV2UKIwEd29yZJRoSUsCaA6JZWVzaDdLAXWMCmF0dHJpYnV0ZXOUKIwF cmVhbG2UXZQojAVCZWluZ5SMBURvaW5nlIwHU2Vuc2luZ5RlhpSMB3Byb2Nlc3OUXZQojApCZWhh dmlvcmFslIwLRXhpc3RlbnRpYWyUjAhNYXRlcmlhbJSMBk1lbnRhbJSMClJlbGF0aW9uYWyUjAZW ZXJiYWyUZYaUjAZwcm9zdWKUXZQojApCZWhhdmlvcmFslIwLRXhpc3RlbnRpYWyUjAhNYXRlcmlh bJSMEU1lbnRhbDogQ29nbml0aXZllIwUTWVudGFsOiBEZXNpZGVyYXRpdmWUjA9NZW50YWw6IEVt b3RpdmWUjBNNZW50YWw6IFBlcnNwZWN0aXZllIwKUmVsYXRpb25hbJSMBlZlcmJhbJRlhpSMA3N1 YpRdlCiMDUFjdHMgdmVyYmFsbHmUjA9Cb2RpbHkgcG9zdHVyZXOUjA5DaXJjdW1zdGFudGlhbJSM CUNvZ25pdGl2ZZSMCENyZWF0aXZllIwMRGVzaWRlcmF0aXZllIwLRGlzcG9zaXRpdmWUjAdFbW90 aXZllIwLRXhpc3RlbnRpYWyUjAlJbnRlbnNpdmWUjAtQZXJzcGVjdGl2ZZSMDVBoeXNpb2xvZ2lj YWyUjApQb3NzZXNzaXZllIwKVmVyYmFsIGFjdJRlhpR0lIwFbWV0YXOUjAR3b3JklEsDhpSFlIwK Y2xhc3NfdmFyc5QpjBJtb2RpZmllZF92YXJpYWJsZXOUXZR1YmF1Lg== </properties> <properties node_id="3" format="literal">{'auto_commit': True, 'color_by_class': True, 'controlAreaVisible': True, 'savedWidgetGeometry': b'\x01\xd9\xd0\xcb\x00\x03\x00\x00\x00\x00\x03\xa7\x00\x00\x02\x8e\x00\x00\x06&amp;\x00\x00\x04m\x00\x00\x03\xa7\x00\x00\x02\x8e\x00\x00\x06&amp;\x00\x00\x04m\x00\x00\x00\x00\x00\x00\x00\x00\rp\x00\x00\x03\xa7\x00\x00\x02\x8e\x00\x00\x06&amp;\x00\x00\x04m', 'select_rows': True, 'show_attribute_labels': True, 'show_distributions': False, 'stored_selection': {'rows': [], 'columns': []}, 'stored_sort': [], '__version__': 1}</properties> <properties node_id="4" format="pickle">gASVEwEAAAAAAAB9lCiMB2FsbF9wb3OUiYwKYXV0b2NvbW1pdJSIjBJjb250cm9sQXJlYVZpc2li bGWUiIwGY3VzdG9tlImMBW5vdW5zlImME3NhdmVkV2lkZ2V0R2VvbWV0cnmUQ0IB2dDLAAMAAAAA A14AAAC8AAAJ8QAABkAAAANeAAAA4QAACfEAAAZAAAAAAAAAAAANcAAAA14AAADhAAAJ8QAABkCU jBJzZWxlY3RlZF9kb2N1bWVudHOUj5QoSxuQjAtzaG93X3Rva2Vuc5SJjAVzdWJqc5SJjAh0YWdf dHlwZZRLAYwIemVyb19wb3OUiYwLX192ZXJzaW9uX1+USwKMEGNvbnRleHRfc2V0dGluZ3OUXZR1 Lg== </properties> </node_properties> <session_state> <window_groups /> </session_state> </scheme> 
Version:
3.36.2
Environment:
Python 3.9.13 on Linux 5.15.0-100-generic #110~20.04.1-Ubuntu SMP Tue Feb 13 14:25:03 UTC 2024 x86_64
Installed Packages:
AnyQt==0.2.0, Babel==2.14.0, Bottleneck==1.3.8, DHTMLParser3==3.0.17, Jinja2==3.1.3, MarkupSafe==2.1.5, Orange3-Network==1.9.0, Orange3-Text==1.15.0, Orange3==3.36.2, PyQt5-Qt5==5.15.2, PyQt5-sip==12.13.0, PyQt5==5.15.10, PyQt6-Qt6==6.6.2, PyQt6-sip==13.6.0, PyQt6==6.6.1, PyQtWebEngine-Qt5==5.15.2, PyQtWebEngine==5.15.6, PyYAML==6.0.1, QtPy==2.4.1, SecretStorage==3.3.3, XlsxWriter==3.2.0, alabaster==0.7.16, annotated-types==0.6.0, anyio==4.3.0, asttokens==2.4.1, attrs==23.2.0, baycomp==1.0.3, beautifulsoup4==4.12.3, biopython==1.83, blis==0.7.11, catalogue==2.0.10, catboost==1.2.3, cattrs==23.2.3, certifi==2024.2.2, cffi==1.16.0, chardet==5.2.0, charset-normalizer==3.3.2, click==8.1.7, cloudpathlib==0.16.0, comm==0.2.1, commonmark==0.9.1, confection==0.1.4, conllu==4.5.3, contourpy==1.2.0, coverage==7.4.3, cryptography==42.0.5, cycler==0.12.1, cymem==2.0.8, debugpy==1.8.1, decorator==5.1.1, defusedxml==0.7.1, dictdiffer==0.9.0, docutils==0.20.1, docx2txt==0.8, en-core-web-sm==3.7.1, et-xmlfile==1.1.0, exceptiongroup==1.2.0, executing==2.0.1, filelock==3.13.1, fonttools==4.49.0, fsspec==2024.2.0, gensim==4.3.2, graphviz==0.20.1, h11==0.14.0, httpcore==1.0.4, httpx==0.27.0, huggingface-hub==0.21.3, idna==3.6, imagesize==1.4.1, importlib-metadata==7.0.1, importlib-resources==6.1.2, ipykernel==6.29.3, ipython==8.18.1, jaraco.classes==3.3.1, jedi==0.19.1, jeepney==0.8.0, jellyfish==1.0.3, joblib==1.3.2, jupyter-client==8.6.0, jupyter-core==5.7.1, keyring==24.3.1, keyrings.alt==5.0.0, kiwisolver==1.4.5, langcodes==3.3.0, langdetect==1.0.9, lemmagen3==3.5.1, matplotlib-inline==0.1.6, matplotlib==3.8.3, more-itertools==10.2.0, mpmath==1.3.0, murmurhash==1.0.10, nest-asyncio==1.6.0, networkx==3.2.1, nl-core-news-sm==3.7.0, nltk==3.8.1, numpy==1.26.4, nvidia-cublas-cu12==12.1.3.1, nvidia-cuda-cupti-cu12==12.1.105, nvidia-cuda-nvrtc-cu12==12.1.105, nvidia-cuda-runtime-cu12==12.1.105, nvidia-cudnn-cu12==8.9.2.26, nvidia-cufft-cu12==11.0.2.54, nvidia-curand-cu12==10.3.2.106, nvidia-cusolver-cu12==11.4.5.107, nvidia-cusparse-cu12==12.1.0.106, nvidia-nccl-cu12==2.19.3, nvidia-nvjitlink-cu12==12.3.101, nvidia-nvtx-cu12==12.1.105, oauthlib==3.2.2, odfpy==1.4.1, openTSNE==1.0.1, openpyxl==3.1.2, orange-canvas-core==0.1.35, orange-widget-base==4.22.0, owlready2==0.45, packaging==23.2, pandas==2.2.1, parso==0.8.3, pexpect==4.9.0, pillow==10.2.0, pip==22.0.4, platformdirs==4.2.0, plotly==5.19.0, preshed==3.0.9, prompt-toolkit==3.0.43, psutil==5.9.8, ptyprocess==0.7.0, pure-eval==0.2.2, pybind11==2.11.1, pycparser==2.21, pydantic-core==2.16.3, pydantic==2.6.3, pydot==2.0.0, pygments==2.17.2, pyparsing==3.1.1, pypdf==4.1.0, pyqtgraph==0.13.3, python-dateutil==2.9.0.post0, python-louvain==0.16, pytz==2024.1, pyzmq==25.1.2, qasync==0.27.1, qtconsole==5.5.1, rapidfuzz==3.6.1, recommonmark==0.7.1, regex==2023.12.25, requests-cache==1.2.0, requests-oauthlib==1.3.1, requests==2.31.0, safetensors==0.4.2, scikit-learn==1.3.2, scipy==1.12.0, segtok==1.5.11, serverfiles==0.3.1, setuptools==58.1.0, shapely==2.0.3, simhash==2.1.2, six==1.16.0, smart-open==6.4.0, sniffio==1.3.1, snowballstemmer==2.2.0, soupsieve==2.5, spacy-legacy==3.0.12, spacy-loggers==1.0.5, spacy==3.7.4, sphinx==7.2.6, sphinxcontrib-applehelp==1.0.8, sphinxcontrib-devhelp==1.0.6, sphinxcontrib-htmlhelp==2.0.5, sphinxcontrib-jsmath==1.0.1, sphinxcontrib-qthelp==1.0.7, sphinxcontrib-serializinghtml==1.1.10, srsly==2.4.8, stack-data==0.6.3, storynavigator==0.0.11, sympy==1.12, tabulate==0.9.0, tenacity==8.2.3, textblob-nl==0.0.1, textblob==0.18.0.post0, thefuzz==0.22.1, thinc==8.2.3, threadpoolctl==3.3.0, tokenizers==0.15.2, torch==2.2.1, tornado==6.4, tqdm==4.66.2, traitlets==5.14.1, transformers==4.38.2, trimesh==4.1.7, triton==2.2.0, tweepy==4.14.0, typer==0.9.0, typing-extensions==4.10.0, tzdata==2024.1, ufal.udpipe==1.3.1.1, url-normalize==1.4.3, urllib3==2.2.1, wasabi==1.1.2, wcwidth==0.2.13, weasel==0.3.4, wikipedia==1.4.0, xgboost==2.0.3, xlrd==2.0.1, yake==0.4.8, zipp==3.17.0
Machine ID:
aa7e8c73-d204-4802-a437-067c75a3dffd
Stack Trace:
Traceback (most recent call last):
  File "/home/flavio/repositories/projects/navigating-stories/orange-story-navigator/orangecontrib/storynavigation/widgets/OWSNActorAnalysis.py", line 763, in selection_changed
    self.show_docs()
  File "/home/flavio/repositories/projects/navigating-stories/orange-story-navigator/orangecontrib/storynavigation/widgets/OWSNActorAnalysis.py", line 803, in show_docs
    self.story_elements_dict[str(c_index)]
KeyError: '27'

```

##### Steps to reproduce the behavior
- `File` widget: `dutch_halliday_action_list.csv`
- `Import Documents` widget: subset of corona stories
- `Elements` widget: language `nl`, ~~custom tags `realm`~~ custom tags `word`, number of segments `1`

##### Additional info (worksheets, data, screenshots, ...)
![image](https://github.com/navigating-stories/orange-story-navigator/assets/55716026/5dd3321a-bcaa-40f2-b243-99da75ceeb71)

**Comment by f-hafner** [comment]
I suspect it has to do with the input text, since the same problem occurs for the actions widget

**Comment by f-hafner** [comment]
stack tracke
```bash
Traceback (most recent call last):
  File "/home/flavio/repositories/projects/navigating-stories/orange-story-navigator/orangecontrib/storynavigation/widgets/OWSNActorAnalysis.py", line 763, in selection_changed
    self.show_docs()
  File "/home/flavio/repositories/projects/navigating-stories/orange-story-navigator/orangecontrib/storynavigation/widgets/OWSNActorAnalysis.py", line 803, in show_docs
    self.story_elements_dict[str(c_index)]
KeyError: '16'
```

next steps: print keys of story elements and see who and why the keys are changing, and why the key 16 is missing.

**Comment by kodymoodley** [comment]
We need to add debug points / print statements to check how the value of ```self.story_elements_dict``` changes. I think that's where the problem is.

**Comment by kodymoodley** [comment]
Should be fixed in #58.
